### PR TITLE
Updated readme file

### LINKED ISSUE #200: Update readme.md
Readme should reflect how to install, start, stop, configure and debug papiea. Should also describe the different env variables that may be used by the various docker-compose files.

### LINKED ISSUE #219: Generate new secret on each papiea run

### PR DESCRIPTION
**Pull request by Emulebest**
resolves #200 
closes #219

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,16 +7,41 @@ between intended state and real world state.
...
+
+# Environment Papiea-Engine
+A set of these variables might be used to tweak the default papiea-engine configuration
+* `MONGO_HOST` - mongo host (default `mongo`)
+* `MONGO_PORT` - mongo port (default `27017`)
+* `PAPIEA_PUBLIC_ADDR` - external papiea address (default `"http://localhost:3000"`)
+* `HOT_RELOAD` - use nodemon to autoreload papiea on code changes (default `false`)
+* `TOKEN_SECRET` - secret for papiea JWT HMAC
+* `TOKEN_EXPIRES_SECONDS` - token expiration time in seconds (default `604800` 1 week)
+* `DEBUG_LEVEL` - log level for [morgan](https://github.com/expressjs/morgan) logging (default `common`)
```
[the remark this answers is not in the document]
Comment (shlomi-nx) [reply README.md]: https://github.com/nutanix/papiea-js/issues/211

File: README.md [hunk 2]
```diff
@@ -7,16 +7,41 @@ between intended state and real world state.
...
+# Debug configuration Papiea-Engine
+Instead of using usual `docker-compose up` in papiea-engine dir, use `docker-compose -f docker-compose-debug.yml` to enable hot-reload and debugger listening on port `9229`
+
+# Environment Papiea-Engine
+A set of these variables might be used to tweak the default papiea-engine configuration
+* `MONGO_HOST` - mongo host (default `mongo`)
+* `MONGO_PORT` - mongo port (default `27017`)
+* `PAPIEA_PUBLIC_ADDR` - external papiea address (default `"http://localhost:3000"`)
+* `HOT_RELOAD` - use nodemon to autoreload papiea on code changes (default `false`)
+* `TOKEN_SECRET` - secret for papiea JWT HMAC
```
[the remark this answers is not in the document]
Comment (Emulebest) [reply README.md]: I didn't want to include a default value for social security reasons, will describe on a call

File: README.md [hunk 3]
```diff
@@ -7,16 +7,41 @@ between intended state and real world state.
...
+* `TOKEN_EXPIRES_SECONDS` - token expiration time in seconds (default `604800` 1 week)
+* `DEBUG_LEVEL` - log level for [morgan](https://github.com/expressjs/morgan) logging (default `common`)
+* `DISALLOW_EXTRA_PROPERTIES` - prohibit usage of props not specified in yaml structure (default `true`)
```
[the remark this answers is not in the document]
Comment (Emulebest) [reply README.md]: I will make an issue for that

[the remark this answers is not in the document]
Comment (Emulebest) [reply README.md]: #213

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Emulebest) [reply README.md]: Let's provide a random value each time the papiea restarts

[the remark this answers is not in the document]
Comment (shlomi-nx) [reply README.md]: Is this backed by an issue? If not please make one and link here. 

The value should be random unless it is specified.

File: README.md [hunk 4]
```diff
@@ -7,16 +7,41 @@ between intended state and real world state.
...
+Papiea-Engine will now be running on port `3000` inside the container and will use port `3333` externally
+
+To stop a container
+
+1. `docker-compose stop`
+
+# Debug configuration Papiea-Engine
+Instead of using usual `docker-compose up` in papiea-engine dir, use `docker-compose -f docker-compose-debug.yml` to enable hot-reload and debugger listening on port `9229`
+
+# Environment Papiea-Engine
```
[the remark this answers is not in the document]
Comment (shlomi-nx) [reply README.md]: Please add a comment here about how to set up the admin service key.

**Comment by Emulebest** [comment]
Papiea Secret random generation issue #219

**Pull request merged**